### DISCUSSION TITLE
Are you any progress in this project?

### DISCUSSION THREAD

**Discussion by BCFCODE** [Sepehrsalimian/WebStarter#16] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/bf4d05d4-97c4-4677-b3f4-97a054f21950)
Have you made any progress on the project? Are you pushing your changes to GitHub? I’m not seeing any updates! @Sepehrsalimian

**Comment by BCFCODE** [comment]
Hey! Just a quick reminder: it’s super important to commit and push your changes as you go. That way, I can see how quickly you’re progressing and give you some feedback. Let’s keep this journey fun and collaborative! 
@Sepehrsalimian #Task

**Comment by BCFCODE** [comment]
![image](https://github.com/user-attachments/assets/0b4d34e3-d821-4598-81c7-40ed933ae8ed)
I pulled your changes and noticed that you only have 2 commits today. Since September 13th, you haven’t made any commits. It seems like you might be struggling to understand the Git workflow. Would you like some tips and an exercise from my side? Because it doesn’t work like this…

Would you like some tips on improving your Git workflow? I can also suggest some exercises to help you get more comfortable with it.
@Sepehrsalimian

**Reply by Sepehrsalimian** [reply, marked as answer]
I agree with you, it's good idea to settle in my mind.
@BCFCODE
